src/sst/elements/memHierarchy/memoryController.cc

```diff
@@ -601,6 +612,12 @@ void MemController::writeData(MemEvent* event) {
         }
         
         backing_->set(addr, event->getSize(), event->getPayload());
+        if (is_debug_event(event)) {
+            Debug(_L10_, "\tContents ");
```

Comment:
Can you change to _L11_? The rest of memH prints values if debug is 11 or higher so it'll be consistent.